tests/ha/test_pod_failure.py

```diff
@@ -1789,8 +1789,12 @@ def test_mpu_after_safe_pod_shutdown(self):
         sysutils.remove_file(self.multipart_obj_path)
         sysutils.remove_file(download_path)
 
-        LOGGER.info("Step 7: Create new bucket and multipart upload and then download 5GB object")
-        bucket_name = "mp-bkt-{}".format(self.random_time)
+        LOGGER.info("Step 7: Multipart upload and then download 5GB object")
+        if CMN_CFG["dtm0_disabled"]:
+            LOGGER.info("Creating new bucket for new upload")
+            bucket_name = "mp-bkt-{}".format(self.random_time)
```

Comment:
better to use perf_counter_ns(), as we have taken this value in setup_method, it will remain same all the time. THis might cause the failures